src/scripting/Scripting.cpp

```diff
@@ -599,11 +609,45 @@ void Scripting::TriggerOnOverlayClose() const
     m_store.TriggerOnOverlayClose();
 }
 
+sol::object Scripting::GetModNames() const
+{
+    auto lockedState = m_lua.Lock();
+    sol::table res(lockedState.Get(), sol::create);
+    for (const auto& name : m_store.GetModNames())
+        res.add(name);
+    return res;
+}
+
 sol::object Scripting::GetMod(const std::string& acName) const
 {
     return m_store.GetMod(acName);
 }
 
+sol::object Scripting::GetModInputHandlers(const std::string& acName) const
+{
+    const auto* modBinds = m_store.GetBinds(acName);
+    if (!modBinds)
+        return sol::nil;
+
+    auto lockedState = m_lua.Lock();
+    const auto& bindings = m_store.GetBindings();
+    sol::table res(lockedState.Get(), sol::create);
+    for (const auto& bind : *modBinds)
+    {
+        sol::table info(lockedState.Get(), sol::create);
+        info["slug"] = bind.ID;
+        info["label"] = bind.DisplayName;
+        info["callback"] = bind.VmHandler;
+        if (std::holds_alternative<std::string>(bind.Description))
+        {
+            info["description"] = std::get<std::string>(bind.Description);
+        }
+        info["bind"] = bindings.GetBindString({acName, bind.ID});
```

Comment:
You cant really get valuable data sometimes from this, CET doesnt handle properly some extended keys, writting them as Unknown into string. + Im a bit unsure what would be the use-case for this bit, or anything here in general. Some sample mod/mods showing how this should work in practics would be nice.

Reply:
Ok, `GetBindString` is not really what I need. I guess I'll change it to
```info["isBound"] = bindings.IsBound({acName, bind.ID});```
to be able to just check if input is bound in CET or not and is it safe to call from my mod.
I am still experimenting with input to be honest, but I think I can show some quick and dirty mod, that shows what I want to do with this functionality.

Reply:
would be nice to see some expected usage, ideally with fixes. maybe sth can be done nicer/better, main point why i wanted some sample :)

Reply:
I created a simple [mod](https://github.com/Supremist/BetterControl) for this. It enables to use Q end E for lean_left/lean_right while aiming. And when player is not aiming - the buttons do default actions (so Q does quick melee)

Reply:
Thanks, will try it with the PR when I get home to see how things work/look in practise 👍